src/components/Filter.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+export default class Filter extends React.Component {
+
+state = {
+    products: [],
+    inputValueMin: '',
+    inputValueMax: '',
+    inputCategory: ''
+}
+
+componentDidMount() {
+    this.props.products
+    this.setState({products: this.state.products})
```

Comment:
aqui pode excluir a linha 13 e na linha 14 na atualização do state, products: this.props.products